examples/using-contentful/gatsby-config.js

```diff
@@ -3,11 +3,22 @@ module.exports = {
     title: `Gatsby with Contentful`,
   },
   plugins: [
+    // {
```

Comment:
Do we need these comments?

Reply:
Added TO-DO in PR description to remove those changes later on - for now I use this to validate everything works when there is 0 content (schema is the same regardless if there is content or not, so queries do not break)